src/drive/web/modules/public/PublicToolbar.jsx

```diff
@@ -147,26 +165,34 @@ class PublicToolbar extends React.Component {
           onDownload={this.downloadFiles}
           discoveryLink={discoveryLink}
           redirectTo={this.redirectTo}
+          isFile={isFile}
         />
       )
     } else if (renderInBar) {
       return (
         <CozybarToolbar
           onDownload={this.downloadFiles}
           discoveryLink={discoveryLink}
+          isFile={isFile}
         />
       )
     } else {
       return (
         <DesktopToolbar
           onDownload={this.downloadFiles}
           discoveryLink={discoveryLink}
+          isFile={isFile}
         />
       )
     }
   }
 }
-
+PublicToolbar.propTypes = {
+  isFile: PropTypes.bool.isRequired,
+  renderInBar: PropTypes.bool,
+  breakpoints: PropTypes.object.isRequired,
+  files: PropTypes.array.isRequired
```

Comment:
same here and same for the shape of the `breakpoints` object above

Reply:
When cozy-ui will expose its own propTypes. Currently it's not the case and I don't want to put that kind of work in this PR.